build-system/pr-check/local-tests.js

```diff
@@ -45,6 +45,7 @@ function main() {
     timedExecOrDie('gulp integration --nobuild --headless --coverage');
     timedExecOrDie('gulp unit --nobuild --headless --coverage');
     timedExecOrDie('gulp codecov-upload');
+    timedExecOrDie('gulp report-test-results');
```

Comment:
Let's do this in a separate PR, including adding the `--report` flag.

Reply:
Done.